data/schema.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS `interview_match` (
+  `guild_id` VARCHAR(20) PRIMARY KEY,
+  `author_id` VARCHAR(20) NOT NULL,
+  `channel_id` VARCHAR(20) NOT NULL,
+  `day_of_the_week` INTEGER NOT NULL,
+  `emoji` TEXT,
```

Comment:
Why emoji is text, are you expecting multiple emojis?

Reply:
Oh, nice catch, the emojis are not exactly chars but I think a varchar(2) should be enough